sse-common/src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java

```diff
@@ -248,14 +235,16 @@ public void run() {
     private void publishStateEvent(SSEChannel.Event event, Message additional) {
         // Only publish these events if we're running
         // in a test.
-        if (!Util.isTestEnv()) {
+        if (!Util.isTestEnv(null)) {
             return;
         }
 
         try {
             SimpleMessage message = new SimpleMessage()
                     .setChannelName("sse")
+                    .set("jenkins_channel", "sse")
```

Comment:
A lot of special tweaks like this were needed to avoid the need to update the client.